### ISSUE TITLE
Can not run -copy separately

### ISSUE THREAD

**Issue by Ossssip** [jeffaco/duplicacy-util#25]
Now having `quote` and pruning options (again, thanks a lot for the fast and kind response), I have created appropriate configs with rich options and tried to use them.
My preference is not to use `duplicacy-util -a`, but separately use backup, copy, check and prune operations - I would like to call prune before copy to decrease data transfer overhead and also to call these operations less frequently than backups.
I have found that `-copy` command actually does not work for me:
```
PS D:\Tools> D:\Tools\duplicacy\duplicacy-util.exe -sd D:\Tools\duplicacy\duplicacy-util\ -f de_tools.yaml -copy
21:04:20 Using global config: D:\Tools\duplicacy\duplicacy-util\duplicacy-util.yaml
21:04:20 Using config file:   D:\Tools\duplicacy\duplicacy-util\de_tools.yaml
Error: No operations to perform (specify -b, -p, -c, or -a)
```
If I call `duplicacy-util -a` for the same config, it does backup, copy, check and prune normally.

**Comment by jeffaco** [comment]
`duplicacy-util -version` please? I think this is a minor bug but did want to verify that.

**Comment by jeffaco** [comment]
I see the problem, resolved in commit 4c1885f
